Networking/Networking/Remote/CardReaderCapableRemote.swift

```diff
@@ -0,0 +1,6 @@
+public protocol CardReaderCapableRemote {
```

Comment:
Can this declaration been moved to the context of CardPresentPaymentsStore? Declaring this protocol in Networking as public seems unnecessary.

If this could be declared in the same file as the store, conformance also could be declared in the same file as: `extension StripeRemote: CardReaderCapableRemote {}`

Reply:
> If this could be declared in the same file as the store, conformance also could be declared in the same file as: extension StripeRemote: CardReaderCapableRemote {}

OMG that's a good idea. I keep forgetting Swift allows that sort of thing

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5922/commits/21297284bab6e6798ad448e49c02dc29fc2816de